src/item/components/modals/AddToCollectionModal.tsx

```diff
@@ -46,6 +45,7 @@ import { getThumbnailForCollection } from '../../../shared/services/stills-servi
 import toastService from '../../../shared/services/toast-service';
 
 import './AddToCollectionModal.scss';
+import FlowPlayerWrapper from '../../../shared/components/FlowPlayerWrapper/FlowPlayerWrapper';
```

Comment:
Import order.

Reply:
fixed in: feature/bookmarks-in-workspace